cms/envs/common.py

```diff
@@ -155,7 +155,10 @@
     'DASHBOARD_SHARE_SETTINGS': {
         # Note: Ensure 'CUSTOM_COURSE_URLS' has a matching value in lms/envs/common.py
         'CUSTOM_COURSE_URLS': False
-    }
+    },
+
+    # Teams feature
+    'ENABLE_TEAMS': False,
```

Comment:
Unfortunately, I don't think that works for feature flags like this one. We could try importing all of `FEATURES` and then having `cms.envs.common.py` explicitly set the other flags it wants, but that is somewhat risky.